sdk/src/main/java/io/radar/sdk/util/RadarFileStorage.kt

```diff
@@ -0,0 +1,77 @@
+package io.radar.sdk.util
+import android.content.Context
+import java.io.File
+import java.io.FileInputStream
+import java.io.FileNotFoundException
+import java.io.FileOutputStream
+
+class RadarFileStorage(private val context: Context) {
+    fun writeData(subDir: String = "",  filename: String, content: String) {
+        var fileOutputStream: FileOutputStream
+        try {
+            if(subDir.isNotEmpty()) {
+                val directory = File(context.filesDir, subDir)
+                directory.mkdirs()
+                val file = File(directory, filename)
+                fileOutputStream = FileOutputStream(file)
+                fileOutputStream.use { it.write(content.toByteArray()) }
+            }
+            else {
```

Comment:
Is the `else` case unused?  Alternatively, I think the `if` case could also handle this case?

Reply:
not as it is, butI agree this chunk can be tidied up. Will clean up the code.